File: app/src/main/java/com/bluetriangle/android/demo/java/screenTracking/SecondTabFragment.java
package com.bluetriangle.android.demo.java.screenTracking;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.bluetriangle.android.demo.R;

public class SecondTabFragment extends Fragment {
    public static SecondTabFragment newInstance() {
        return new SecondTabFragment();
    }

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_second_tab, container, false);
    }
}

File: app/src/main/java/com/bluetriangle/android/demo/groupingpoc/tabs/SecondTabFragment.kt
package com.bluetriangle.android.demo.groupingpoc.tabs

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.compose.ui.platform.ComposeView
import androidx.fragment.app.Fragment
import com.bluetriangle.android.demo.R
import com.bluetriangle.android.demo.compose.TabContent

class SecondTabFragment : Fragment() {

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.fragment_second_tab_grouping, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {

        view.findViewById<ComposeView>(R.id.composeContainer).setContent {
            TabContent(tabName = "ComposeTab")
        }
    }
}